components/yaml/introduction.rst

```diff
@@ -250,6 +233,77 @@ representation of the object.
     parsers will likely not recognize the ``php/object`` tag and non-PHP
     implementations certainly won't - use with discretion!
 
+Handling Invalid Types
```

Comment:
before this headline, we should add a line: `.. _invalid-types-and-object-serialization:` This way, all section links still work across versions.